webpack.config.js

```diff
@@ -61,16 +63,27 @@ module.exports = [{
           presets: ['es2015', 'react'],
         },
       },
+
+      /*
+       * Parse SCSS to minified CSS, then postprocess with postcss autoprefixer plugin
+       */
       {
         test: /\.scss$/,
-        loader: ExtractTextPlugin.extract(['css','sass']),
+        loader: ExtractTextPlugin.extract('style', ['css?minimize', 'postcss', 'sass']),
```

Comment:
The CSS currently minifies, yes. If the order is different, `postcss` throws an error because it attempts to interpret SASS (which it can't do). From what I could understand, webpack seems to compile these in reverse order from how they are listed in `ExtractTextPlugin`. In other words, it first converts Sass to CSS, then the auto-prefixing happens, then `css-loader` takes over the preprocessed, autoprefixed file. This is why it was so complicated to implement.